classes/test/NioFilesPaths.java

```diff
@@ -0,0 +1,179 @@
+package classes.test;
+
+import java.io.*;
+import java.nio.file.*;
+import java.nio.file.attribute.FileTime;
+import java.util.Arrays;
+
+class NioFilesPaths {
+  static void printFile(Path fPath) throws IOException {
+    Files.lines(fPath).forEach(System.out::println);
+  }
+
+  private static FileSystem dfs = FileSystems.getDefault();
+
+  public static void main(String[] args) throws IOException {
+    final String testDir = "./classes/test/data/FileOps";
+    final Path testDirPath = Paths.get("./classes/test/data/FileOps");
+    // I like scopes.
+    {
+      // This file does not exist.
+      final File f = Paths.get("/dfsd/dsfds").toFile();
```

Comment:
For this test, the `Path.toFile()` is the important bit. To check if the resulting `File` exists we have to use java.io method, can't use nio there.